shopping-cart/shopping-cart-java/shopping-cart/src/test/java/com/example/shoppingcart/impl/ShoppingCartServiceTest.java

```diff
@@ -43,7 +43,7 @@ public void shouldAddAnItem() {
         String cartId = randomId();
         String itemId = randomId();
 
-        Pair<ResponseHeader, Done> result = Await.result(shoppingCartService.addItem(cartId).withResponseHeader().invoke(new ShoppingCartItem(itemId, 2)));
+        Pair<ResponseHeader, ShoppingCartView> result = Await.result(shoppingCartService.addItem(cartId).withResponseHeader().invoke(new ShoppingCartItem(itemId, 2)));
```

Comment:
Maybe add some assertions based on the `ShoppingCartView`?

Reply:
Yes, I will do it.

Reply:
done